docs/lite.md

```diff
@@ -17,13 +17,13 @@ The Pluggable Modular Parsers provide the ability to customize and extend the da
 The Splunk platform is the destination for the syslog data. Splunk allows for comprehensive analysis, searching, and visualization of the processed data.
```

Comment:
SC4S sends your syslog data to the Splunk platform where you can perform comprehensive analysis, searching, and visualization of the processed data.